varbase_layout_builder.libraries.yml

```diff
@@ -37,3 +37,45 @@ vlb-colors:
   css:
     theme:
       css/theme/vlb-colors.theme.css: {}
+
+# Layout builder modal claro theme 
+claro:
+  version: VERSION
+  css:
+    base:
```

Comment:
All these css files should have paths similar to Claro's folder structure/paths. 
For example:
```
 css/claro/accordion.css: {}
 css/claro/action-link.css: {}
```

should be:

```
 css/claro/components/accordion.css: {}
 css/claro/components/action-link.css: {}
```

This will make it easier to maintain.